docs/components/plugins/Redirect.tsx

```diff
@@ -14,11 +7,7 @@ const Redirect: React.FC<{ path: string }> = ({ path }) => {
     }, 0);
   });
 
-  return (
-    <div css={CONTAINER_STYLE}>
-      Redirecting to <a href={path}>{path}</a>
-    </div>
-  );
+  return null;
```

Comment:
Im just not familiar: when/where did this component appear?

Reply:
one example of where this is used is https://docs.expo.io/build/ - it redirects to https://docs.expo.io/build/introduction/

Reply:
Yes- it is used only here at the moment